### PR TITLE
"set picture as" with resized images

### PR DESCRIPTION
**Pull request by tobiasKaminsky**
This was somehow removed during merge, so it is now back.

Steps to test:
- set a non-download image as wallpaper

### CODE REVIEW

File: src/main/java/com/owncloud/android/ui/helpers/FileOperationsHelper.java [hunk 1]
```diff
@@ -515,30 +516,34 @@ public void sendCachedImage(OCFile file) {
...
+                intent.setDataAndType(uri, file.getMimetype());
+                mFileActivity.startActivityForResult(Intent.createChooser(intent,
+                        mFileActivity.getString(R.string.set_as)), 200);
 
-                } catch (ActivityNotFoundException exception) {
-                    Toast.makeText(context, R.string.picture_set_as_no_app, Toast.LENGTH_LONG).show();
-                }
+                intent.setDataAndType(uri, file.getMimetype());
+            } catch (ActivityNotFoundException exception) {
+                Toast.makeText(context, R.string.picture_set_as_no_app, Toast.LENGTH_LONG).show();
```
Comment (AndyScherzinger) [inline src/main/java/com/owncloud/android/ui/helpers/FileOperationsHelper.java]: Can we make the Toast a Snackbar? 😃

**Comment by AndyScherzinger** [comment]
👍 Just a minor issue

File: src/main/java/com/owncloud/android/ui/helpers/FileOperationsHelper.java (hunk 1 above)
Comment (tobiasKaminsky) [reply src/main/java/com/owncloud/android/ui/helpers/FileOperationsHelper.java]: Nice catch 👍

**Comment by mario** [comment]
Rebase pls.

**Comment by AndyScherzinger** [comment]
@mario rebased 😃

**Comment by mario** [comment]
👍

**Pull request merged**